vcs/vcsvtk/pipeline1d.py

```diff
@@ -103,7 +104,7 @@ def plot(self, data1, data2, tmpl, grid, transform, **kargs):
                 self._context().canvas.plot(m, donotstoredisplay=True)
 
         ren2 = self._context().createRenderer()
-        ren2.SetLayer(1)
+        self._context().setLayer(ren2, l.priority)
```

Comment:
didn't @danlipsa has this change in his branch? may be merge his branch into yours?